app/client-v2/src/layouts/AppNav.vue

```diff
@@ -52,7 +52,7 @@ const toggle = (event: MouseEvent) => {
       </template>
       <template #end>
         <div class="flex gap-3">
-          <div><Button :icon="themeIcon" @click="toggleTheme" outlined /></div>
+          <div><Button aria-label="theme-switcher" :icon="themeIcon" @click="toggleTheme" outlined /></div>
```

Comment:
`aria-label` values should be human-consumable shouldn't they? Typically read out by a screen reader, so punctuation maybe not an issue, but I'd expect this to be formatted like a label. 
```suggestion
          <div><Button aria-label="Switch theme" :icon="themeIcon" @click="toggleTheme" outlined /></div>
```